gravitee-apim-console-webui/src/management/api/apis.module.ts

```diff
@@ -97,6 +98,7 @@ graviteeManagementModule.run(apiPermissionHook);
     ApiCreationGetStartedModule,
     ApiCreationV2Module,
     ApiCreationV4Module,
+    ApiDocumentationV4Module,
```

Comment:
Does the new documentation only apply to V4? Or will it also be used for V2 api?

Reply:
The new components will be used only for V4 APIs for now. We will not have the same feature scope as for V2 (at first, only markdown pages, and no import from external sources).